test/raft.test.ts

```diff
@@ -0,0 +1,84 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+import { Raft } from '../typechain-types'
+import { Event, Wallet } from 'ethers'
+import { waffle } from 'hardhat'
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers'
+
+describe('Raft', async function () {
+    let raftContract: Raft
+    const name = 'Raft'
+    const symbol = 'RAFT'
+    const chainId = 31337
+    const tokenURI = 'blah'
+
+    async function deployContractFixture() {
```

Comment:
why not leverage `before()`?

Reply:
I found this method in the Hardhat testing docs here: https://hardhat.org/tutorial/testing-contracts#reusing-common-test-setups-with-fixtures